TechnicalTask-lvl1/AllUsersModule/UserTableViewCell.swift

```diff
@@ -0,0 +1,132 @@
+    static let boldFont: UIFont = .boldSystemFont(ofSize: 18)
+    static let commonInset: CGFloat = 8.0
+}
+
+class UserTableViewCell: UITableViewCell, Reusable {
+    // MARK: - GUI Properties
+    private let userNameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.font = Constants.userNameFont
+        label.text = "User Name"
+        return label
+    }()
+    
+    private let userEmailLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = Constants.userEmailFont
+        label.text = "User email"
+        return label
+    }()
+    
+    private let addressLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = Constants.addressFont
+        label.text = "Address:"
+        return label
+    }()
+    
+    private let cityStreetLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        label.font = Constants.cityStreetFont
+        label.text = "City name,\nstreet name"
+        return label
+    }()
+    
+    private let containerView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.layer.borderWidth = 1
+        view.layer.borderColor = UIColor.black.cgColor
+        view.layer.cornerRadius = 8
+        view.layer.masksToBounds = true
+        return view
+    }()
+    
+    // MARK: - Life cycle
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        selectionStyle = .none
+        setupUI()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Methods
+    func configure(with user: UserModel) {
+        userNameLabel.text = user.name
+        userEmailLabel.text = user.email
+        
+        var addressText = ""
+        if let city = user.address?.city, !city.isEmpty {
+            addressText += city
+        }
+        
+        if let street = user.address?.street, !street.isEmpty {
+            if !addressText.isEmpty {
+                addressText += ",\n"
+            }
+            addressText += street
+        }
+        
+        cityStreetLabel.text = addressText.isEmpty ? "N/A" : addressText
+    }
+}
+
+// MARK: - Private methods
+private extension UserTableViewCell {
+    func setupUI() {
+        contentView.addSubview(containerView)
+        containerView.addSubviews([
+            userNameLabel,
+            userEmailLabel,
+            addressLabel,
+            cityStreetLabel
+        ])
+        
+        NSLayoutConstraint.activate([
```

Comment:
I think it might be better to move the constraints to separate methods and call them in setupUI